### PR TITLE
Add article about exporting domain names to file

### PR DESCRIPTION
**Pull request by dallasread**
This PR adds a new article about exporting domain names to a file.

### CODE REVIEW

File: content/articles/export-domain-names-to-file.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+| Attribute | Description |
+| --- | --- |
+| **domain_name** | name of the domain|
+| **labels** | labels that have been attached to the domain|
+| **registration_state** | the current registration state|
+| **registration_contact** | the current registration contact|
+| **expiration_date** | the date the domain expires |
+| **renew_by** | the date the domain must be renewed by |
+| **auto_renews** | whether the domain auto-renews |
```
Comment (dallasread) [inline content/articles/export-domain-names-to-file.md:44]:
```suggestion
| **auto_renews** | whether the domain auto-renews |
| **renewal_price** | the domain renewal price |
```

File: content/articles/export-domain-names-to-file.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
```
Comment (dallasread) [inline content/articles/export-domain-names-to-file.md:44]:
```suggestion
| **auto_renews** | whether the domain auto-renews |
| **premium_domain** | whether the domain has premium pricing or not |
```

**Review by itsalyse — APPROVED** [review]

File: content/articles/export-domain-names-to-file.md [hunk 3]
```diff
@@ -54,3 +54,7 @@ We include domain registration and DNS zone details in the exported file:
 | **zone_providers** | the integrated zone providers |
 | **certificates_count** | the number of related certificates |
 | **email_forwards_count** | the number of related email forwards |
+
```
Comment (dallasread) [inline content/articles/export-domain-names-to-file.md:57]: Whoops. Not sure how I missed adding this. 🤦‍♂️

**Review by kojdm — APPROVED** [review]
💯

**Review by stephanoodle — APPROVED** [review]

**Review by Pemacaa — APPROVED** [review]

**Pull request merged**